protocol/specification/descriptive.rst

```diff
@@ -127,43 +127,102 @@ Optional Module Properties
     Related issue: :issue:`008 Groups and Hierarchy`
 
 ``"meaning"``
-    A tuple, with the following two elements:
+   An unordered dictionary regarding the module meaning. It provides metadata that is useful for interpreting measurement data in a machine-readable format. It can have the keys ``function``, ``importance``, ``belongs_to``, ``link`` and ``key``, all of which are optional, with some restrictions. A meaning property can also be added on the :ref:`parameter-meaning <paremeter level>`.
 
-    1. A string from an extensible list of predefined meanings:
+   .. note::
+      In order for the meaning dicionary to be valid, it must contain at least a ``"link"`` or a ``"function"`` field.
 
-       - ``"temperature"`` (the sample temperature)
-       - ``"temperature_regulation"`` (to be specified only if different from 'temperature')
-       - ``"magneticfield"``
-       - ``"electricfield"``
-       - ``"pressure"``
-       - ``"rotation_z"`` (counter clockwise when looked at 'from sky to earth')
-       - ``"humidity"``
-       - ``"viscosity"``
-       - ``"flowrate"``
-       - ``"concentration"``
+   - ``"function"`` a string from an extensible list of predefined functions.
 
-       This list may be extended later.
+      Predefined ``"functions"``:
+         * ``"temperature"``
```

Comment:
This needs an empty line between `Predefined functions` and the first item, and the bullets (asterisks) should be on the same level as `Predefined` to avoid a blockquote.